helper/development-container/container-scripts/entrypoint.sh

```diff
@@ -6,6 +6,14 @@ DOCKER_SOCKET=/var/run/docker.sock
 grep -q localhost /etc/hosts || sudo bash -c 'echo "127.0.0.1 localhost" >> /etc/hosts'
 grep -q $(hostname) /etc/hosts || sudo bash -c 'echo "127.0.0.1 $(hostname)" >> /etc/hosts'
 
+# Start needed services
+sudo service rabbitmq-server restart
+sudo service mysql start
+sudo mysql -e "CREATE DATABASE unittest DEFAULT CHARSET utf8mb4 DEFAULT COLLATE utf8mb4_unicode_ci;"
+sudo mysql -e "CREATE USER 'test'@localhost IDENTIFIED BY 'test';"
+sudo mysql -e "GRANT ALL PRIVILEGES ON unittest.* to 'test'@localhost;"
+sudo mysql -e "FLUSH PRIVILEGES;"
+
```

Comment:
Here, only the database for unit tests is created.
We also need to create the database for running the service... but this might be a later process.